qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -0,0 +1,274 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Built-in transpiler stage plugins for preset pass managers."""
+
+from qiskit.transpiler.passmanager_config import PassManagerConfig
+from qiskit.transpiler.passmanager import PassManager
+from qiskit.transpiler.exceptions import TranspilerError
+from qiskit.transpiler.passes import BasicSwap
+from qiskit.transpiler.passes import LookaheadSwap
+from qiskit.transpiler.passes import StochasticSwap
+from qiskit.transpiler.passes import SabreSwap
+from qiskit.transpiler.passes import Error
+from qiskit.transpiler.preset_passmanagers import common
+from qiskit.transpiler.preset_passmanagers.plugin import PassManagerStagePlugin
+
+
+class BasicSwapPassManager(PassManagerStagePlugin):
+    """Plugin class for routing stage with :class:`~.BasicSwap`"""
+
+    def pass_manager(self, pass_manager_config: PassManagerConfig) -> PassManager:
+        """Build routing stage PassManager."""
+        opt_level = pass_manager_config.optimization_level
+        seed_transpiler = pass_manager_config.seed_transpiler
+        target = pass_manager_config.target
+        coupling_map = pass_manager_config.coupling_map
+        backend_properties = pass_manager_config.backend_properties
+        routing_pass = BasicSwap(coupling_map)
+        vf2_call_limit = common.get_vf2_call_limit(
+            opt_level, pass_manager_config.layout_method, pass_manager_config.initial_layout
+        )
+        if opt_level == 0:
+            return common.generate_routing_passmanager(
+                routing_pass,
+                target,
+                coupling_map=coupling_map,
+                seed_transpiler=seed_transpiler,
+                use_barrier_before_measurement=True,
+            )
+        elif opt_level == 1:
```

Comment:
I'm actually surprised pytlint didn't yell at me about this, it's a minor thing, but I'll update this because that's typically what I'd do. in an earlier version I didn't return in the if block and collected into a common variable that returned (I had to add the error to catch some edge cases, which changed the logic here).

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8305/commits/6cb6b2011a6d72bd837bfb94f994e6f97a3ab8a7